tests/checkpoint/test_checkpoint_with_fluent_datasources.py

```diff
@@ -1037,3 +1041,1495 @@ def test_checkpoint_run_adds_validation_ids_to_expectation_suite_validation_resu
+                            "anonymized_expectation_suite_name": "295722d0683963209e24034a79235ba6",
+                            "anonymized_action_list": [
+                                {
+                                    "anonymized_name": "8e3e134cd0402c3970a02f40d2edfc26",
+                                    "parent_class": "StoreValidationResultAction",
+                                },
+                                {
+                                    "anonymized_name": "40e24f0c6b04b6d4657147990d6f39bd",
+                                    "parent_class": "StoreEvaluationParametersAction",
+                                },
+                                {
+                                    "anonymized_name": "2b99b6b280b8a6ad1176f37580a16411",
+                                    "parent_class": "UpdateDataDocsAction",
+                                },
+                            ],
+                        },
+                        {
+                            "anonymized_expectation_suite_name": "295722d0683963209e24034a79235ba6",
+                            "anonymized_action_list": [
+                                {
+                                    "anonymized_name": "8e3e134cd0402c3970a02f40d2edfc26",
+                                    "parent_class": "StoreValidationResultAction",
+                                },
+                                {
+                                    "anonymized_name": "40e24f0c6b04b6d4657147990d6f39bd",
+                                    "parent_class": "StoreEvaluationParametersAction",
+                                },
+                                {
+                                    "anonymized_name": "2b99b6b280b8a6ad1176f37580a16411",
+                                    "parent_class": "UpdateDataDocsAction",
+                                },
+                            ],
+                        },
+                    ],
+                },
+                "event": "checkpoint.run",
+                "success": False,
+            }
+        )
+    ]
+    # noinspection PyUnresolvedReferences
+    actual_events: List[unittest.mock._Call] = mock_emit.call_args_list
+    assert actual_events == expected_events
+
+    assert len(context.validations_store.list_keys()) == 0
+
+    context.add_expectation_suite("my_expectation_suite")
+    # noinspection PyUnusedLocal
+    result = checkpoint.run(
+        validations=[
+            {"batch_request": batch_request_0},
+            {"batch_request": batch_request_1},
+            {"batch_request": batch_request_2},
+            {"batch_request": batch_request_3},
+        ]
+    )
+
+    assert len(context.validations_store.list_keys()) == 4
+    assert result["success"]
+
+    assert mock_emit.call_count == 14
+
+    # noinspection PyUnresolvedReferences
+    expected_events: List[unittest.mock._Call] = [
+        mock.call(
+            {
+                "event_payload": {
```

Comment:
please shrink this payload definition

Reply:
@donaldheppner Please see `https://github.com/great-expectations/great_expectations/pull/7740#discussion_r1179522567` above.  Thanks.